tests/e2e/manifest_reconciliation_test.go

```diff
@@ -106,5 +107,41 @@ var _ = Describe("Manifest Skip Reconciliation Label", Ordered, func() {
 				WithArguments(defaultRemoteKymaName, RemoteNamespace, runtimeClient, shared.StateReady).
 				Should(Succeed())
 		})
+
+		It("When a blocking finalizer is added to the Manifest CR", func() {
+			Eventually(AddFinalizerToManifest).
```

Comment:
Is there a reason to use `Eventually` here, instead of just failing on error? Are errors expected when adding Finalizers?

Reply:
I used Eventually here and in the below statement because sometimes updating the objects can cause a "conflict" if it was being reconciled at the same time or any other update is happening at the same time